lambdas/web3.ts

```diff
@@ -175,7 +186,7 @@ export const getLatestEventsAndProcess: Handler = async (_event, _context: Conte
   let updateBlockNumberPromises = [];
   web3.eth.getBlockNumber().then(number => {
     contractDbItems.forEach(contract => {
-      updateBlockNumberPromises.push(updateLastBlockChecked(number, contract.address));
+      updateBlockNumberPromises.push(updateLastBlockChecked(number, contract.address, 0));
```

Comment:
Third parameter here is a bit meaningless if you don't know the implementation. `errorTries` is equal to zero, does it means there will be 0 error tries? No, there will be 100.
It is a common design to split a recursive function into two functions—public and internal. There the public function is calling the internal with all required arguments:
```javascript
async function updateLastBlockChecked(blockNumber: number, contractAddress: string) {
  return await _updateLastBlockChecked(blockNumber, contractAddress, 0);
}
```
So you don't have to expose parameters which are used for recursion.